data/themes/CMakeLists.txt

```diff
@@ -2,3 +2,7 @@ INCLUDE(InstallHelpers)
 
 INSTALL_DATA_SUBDIRS("themes" "*.png;*.css")
 
+# Handle custom branding
+INSTALL(FILES "${CMAKE_BINARY_DIR}/cmake/branding/png/splash.png" DESTINATION "${LMMS_DATA_DIR}/themes/default" OPTIONAL)
+INSTALL(FILES "${CMAKE_BINARY_DIR}/cmake/branding/png/icon_32x32.png" DESTINATION "${LMMS_DATA_DIR}/themes/default" RENAME icon_small.png OPTIONAL)
+INSTALL(FILES "${CMAKE_BINARY_DIR}/cmake/branding/png/icon_128x128.png" DESTINATION "${LMMS_DATA_DIR}/themes/default" RENAME icon.png OPTIONAL)
```

Comment:
Upon re-review, these should probably be moved back to `cmake/branding/CMakeLists.txt`.  I had thought it was needed due to an order of operations problem, but after re-reviewing, it was more likely a CMake caching issue.